src/inspirehep-holdingpen-js/holdingpen/holdingpen.controllers.js

```diff
@@ -78,6 +81,23 @@
     function setDecision(id, decision) {
       HoldingPenRecordService.setBatchDecision($scope.vm.invenioSearchResults.hits.hits, [+id], decision);
     }
+
+    function onBestMatchSelected(workflowId){
+      HoldingPenRecordService.setMatchDecision(workflowId, $scope.selectedBestMatchesById[workflowId])
+        .then(function () {
+          $scope.matched = true;
```

Comment:
this will affect the whole result list, it's better to handle it per ``workflowId ``